ta/pkcs11/user_ta.mk

```diff
@@ -1 +1,11 @@
 user-ta-uuid := fd02c9da-306c-48c7-a49c-bbd827ae86ee
+
+all: ci-checks
+
+.PHONY: ci-checks
+ci-checks: pkcs11-ta-ci-verify-helpers
+
+.PHONY: pkcs11-ta-ci-verify-helpers
+pkcs11-ta-ci-verify-helpers:
```

Comment:
I would recommend leaving `ci` out of the name because Continuous Integration (the GitHub PR checks using Azure etc.) is not involved here.

Reply:
OK